spec/features/admin/announcement_management_spec.rb

```diff
@@ -0,0 +1,162 @@
+require 'rails_helper'
+
+describe 'Administration announcement management' do
+  subject { page }
+
+  let!(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    let!(:user) { create(:user, role: :administrator) }
+
+    before do
+      login_as(user, scope: :user)
+    end
+
+    describe 'announcement creation' do
+      before { visit new_admin_announcement_path }
+
+      context 'with invalid information' do
+        before { click_button 'Create' }
+
+        it 'stays on the same page' do
+          expect(page).to have_button('Create')
+        end
+
+        it 'shows errors' do
+          expect(page).to have_css('div.has-error')
+        end
+      end
+
+      context 'with valid information' do
+        let(:announcement) { build(:instance_announcement, instance: instance) }
+
+        before do
+          fill_in 'instance_announcement[title]',      with: announcement.title
+          fill_in 'instance_announcement[content]',    with: announcement.content
```

Comment:
Line is too long. [81/80]<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.